theme/scripts/components/Shapes.ts

```diff
@@ -1,4 +1,4 @@
-import Component from './Component'
+import { Component } from '@mangoweb/scripts-base'
```

Comment:
Shapes je součástí `@mangoweb/scripts-base`, [použít prosím tu](https://github.com/manGoweb/npm/blob/master/scripts-base/src/components/Shapes.ts).

Reply:
Nevim, nějak to nefunguje :(

<img width="1281" alt="Screenshot 2019-07-16 at 09 46 45" src="https://user-images.githubusercontent.com/316435/61275876-23029d00-a7af-11e9-9b94-07162bbbbec1.png">